index.js

```diff
@@ -86,6 +87,26 @@ function fastifyView (fastify, opts, next) {
     })
   }
 
+  function viewNunjucks (page, data) {
+    if (!page || !data) {
+      this.send(new Error('Missing data'))
+      return
+    }
+    // Create nunjucks environment with custom options.
+    // Do not pass templatesDir to nunjucks.configure(),
+    // otherwise all nunjucks function will take templatesDir as their
+    // path, and will cause error in test.
+    // (Maybe a silly bug/feature of nunjucks?)
+    const env = engine.configure(options)
+    // Must use path.join() to dealing with the '/index.njk' in test.js.
+    // All people will think that's represent index.njk under root.
+    // Why the author prefer '/index' to just 'index'? Maybe frontend style?
```

Comment:
Can you remove these comments? :)

Reply:
Yeah I can. :-p I just got puzzled (this word makes me failed many times and changed my code to support this) and not seriously.